lib/smartcar/utils.rb

```diff
@@ -34,7 +34,7 @@ def get_config(config_name)
     #
     # @return [RecursiveOpenStruct]
     def json_to_ostruct(hash)
-      RecursiveOpenStruct.new(hash)
+      RecursiveOpenStruct.new(hash, recurse_over_arrays: true)
```

Comment:
What this additional option does is if there is an array of object in json it recurses over it and converts it into array of objects....This was required to give dot notation access to each capability object inside compatibility v2 response